packages/manager-react-components/src/components/templates/update-name-modal/update-name-modal.component.tsx

```diff
@@ -139,7 +139,12 @@ export const UpdateNameModal: React.FC<UpdateNameModalProps> = ({
         {cancelButtonLabel || t('updateModalCancelButton')}
       </OsdsButton>
       <OsdsButton
-        disabled={isLoading || isPatternError || undefined}
+        disabled={
+          isLoading ||
+          isPatternError ||
+          defaultValue === displayName ||
```

Comment:
**question:** What if user wants to proceed with the displayName itself? i would prefer to accept a function or expression as prop and based on that disable the button.

Reply:
If there is no change, we should not let the user trigger the API
He should cancel the action instead of triggering the API since he doesn't want to change the displayName